samples_project/Assets/SampleViewer/Samples/ThirdPerson/readme.md

```diff
@@ -0,0 +1,33 @@
+# Explore with a Third Person Character
+
+Allows users to explore from the perspective of a Third Person Character.
```

Comment:
```suggestion
Explore from the perspective of a third person camera following a controllable character.
```
We shouldn't us user story type language here. The users are already consuming this as public doc.

Reply:
That makes sense about the tone, I just completely removed this sentence since the title already tells it all